include/base/OpenMCProblemBase.h

```diff
@@ -516,11 +530,17 @@ class OpenMCProblemBase : public CardinalProblem, public PostprocessorInterface
   /// The initial OpenMC seed
   const int64_t _initial_seed;
 
-  /// The index of a global tally to accumulate the scores required for kinetics parameters.
-  int _ifp_tally_index = -1;
+  /// The index of a global tally to accumulate the common scores required for kinetics parameters.
```

Comment:
Can different comments be used here for line 533 and line 536? Right now they are the same